crates/pop-common/src/helpers.rs

```diff
@@ -31,6 +31,18 @@ pub fn replace_in_file(file_path: PathBuf, replacements: HashMap<&str, &str>) ->
 	Ok(())
 }
 
+/// Gets the last component (name of a project) of a path or returns a default value if the path has no valid last component.
+///
+/// # Arguments
+/// * `path` - Location path of the project.
+/// * `default` - The default string to return if the path has no valid last component.
+pub fn get_project_name_from_path(path: &Path, default: &str) -> String {
+	path.file_name()
```

Comment:
path.components().last().map_or() might also be possible. Not sure which is best.

Reply:
Seems to produce the same output as `file_name`, so I'll keep it for now.